tests/model_hub_tests/transformation_tests/generate_model_diffs.py

```diff
@@ -0,0 +1,91 @@
+'''
```

Comment:
We have three terms: "model_diffs" in this script name, "model_diff_ref" in the generated file names, and "pa_reference" in the variable name for the same thing. I would choose one term.

Reply:
Fixed. Aligned all the names